src/module.c

```diff
@@ -11511,18 +11511,6 @@ static uint64_t moduleEventVersions[] = {
  *
  *         int32_t hz;  // Approximate number of events per second.
  *
- * * ValkeyModuleEvent_MasterLinkChange
```

Comment:
Instead of deleting this, can we just update it to `ValkeyModuleEvent_PrimaryLinkChange` ?

Reply:
yeah we need to keep it, just rename it.

@madolson be careful when you say "remove" next time. :laughing:

Reply:
Oke, Well done
I renamed it to `ValkeyModuleEvent_PrimaryLinkChange`